### PR TITLE
Profilo eng Floriana Ferrara Issue #367

### PR DESCRIPTION
**Pull request by simonesurdi**
Fixes issue #367

Brief description of changes proposed in this pull request:

cc @teamdigitale/code-reviewers

### CODE REVIEW

File: en/people/floriana-ferrara.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+period_provided: until September 16, 2018
+annual_compensation: € 80,000
+approved_by_court: September 24, 2017
+---
+I am from Taranto, I earned a degree in Information Science at University of Bari and I am a computer expert. 
+
+I spent two years at CORINTO (National Research Consortium for Objects Technology) in Bari and then I started working for IBM. 
+Here I have held various roles in different fields which are all related to the making of innovative digital projects.
+
+During a business meeting I met Diego and shortly after I became a member of the Digital Transformation Team, then I started the interview for the Technical Project Manager position. Being selected was a great honour and I am grateful to the IBM for having allowed me a year off to embark on this wonderful adventure.
```
Comment (cloudify) [inline en/people/floriana-ferrara.md]: `the IBM` suona male, meglio togliere `the`?

Comment (gvarisco) [reply en/people/floriana-ferrara.md]: +1, 'the IBM '-> 'IBM'

Comment (gvarisco) [inline en/people/floriana-ferrara.md]: The english sentence here doesn't make much sense to me. "During a meeting .. I met $x, and SHORTLY AFTER I became, THEN I started the interview" (?)

File: en/people/floriana-ferrara.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+medium_user: FlorianaFer
+linkedin_url: https://www.linkedin.com/in/filomena-floriana-ferrara-50588912 
+ref: floriana-ferrara
+parent_ref: team
+start_date: September 04, 2017
+period_provided: until September 16, 2018
+annual_compensation: € 80,000
+approved_by_court: September 24, 2017
+---
+I am from Taranto, I earned a degree in Information Science at University of Bari and I am a computer expert.
```
Comment (gvarisco) [inline en/people/floriana-ferrara.md]: "I am", "I earned", "I am" -> I'd rephrase it a bit, eg. "I'm a computer expert originally from Taranto who has earned a degree in Information Science at University of Bari"
